src/TelemetryConsumers/Orleans.TelemetryConsumers.Datadog/Orleans.TelemetryConsumers.Datadog.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <PackageId>Microsoft.Orleans.OrleansTelemetryConsumers.Datadog</PackageId>
```

Comment:
```suggestion
    <PackageId>Microsoft.Orleans.TelemetryConsumers.Datadog</PackageId>
```

Reply:
Yeah, I’m happy to make this change. What about the newrelic consumer? I based my code on it and both share the package id prefix: https://raw.githubusercontent.com/dotnet/orleans/main/src/TelemetryConsumers/Orleans.TelemetryConsumers.NewRelic/Orleans.TelemetryConsumers.NewRelic.csproj